src/boards/mod.rs

```diff
@@ -65,13 +65,6 @@ pub fn get_board(board_id: String) -> Result<Board, Box<dyn Error>> {
 	return Err(From::from("Unknown target board!"));
 }
 
-pub fn get_boot_env(fdt: String) -> String {
```

Comment:
we used this in the past to adjust the fdt on the fly.  We *might* need it again in the future, but indeed it's unused today.

Reply:
Git never forgets... 😄